addons/voice/org.openhab.voice.gtts/.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea/
+.settings/
+bin/
+.project
+target/
```

Comment:
There is a [.gitignore](https://github.com/openhab/openhab2-addons/blob/master/.gitignore) file at the root of the repository which already defines these ignores. Since they are inherited this file can probably be removed.